SPECS/gdb/gdb.spec

```diff
@@ -71,13 +72,15 @@ rm -vf %{buildroot}%{_libdir}/libaarch64-unknown-linux-gnu-sim.a
 %check
 # disable security hardening for tests
 rm -f $(dirname $(gcc -print-libgcc-file-name))/../specs
+
 %make_build check TESTS="gdb.base/default.exp"
 
 %files -f %{name}.lang
 %defattr(-,root,root)
 %license COPYING
 %exclude %{_datadir}/locale
 %exclude %{_includedir}/*.h
+%{_libdir}/libsframe.a
```

Comment:
Is this file necessary to package?

Reply:
Hey @anphel31 I was Umming and ahing on this, i have seen a few Cplus apps that want to use this static library. What are your thoughts if I put it into another package?

Reply:
If you think it's useful, I think it would be best to package it in a gdb-devel subpackage

Reply:
And if you create the gdb-devel package, it looks like a few other files belong there: specifically at least "%{_includedir}/gdb/*.h"

Reply:
If someone is using gdb currently for development, it probably would be - they would need to now also install gdb-devel. It seems like the right thing to do though.

Reply:
Ok, i just ended up dropping it.